pyatlan/model/assets/__init__.py

```diff
@@ -242,6 +252,7 @@
 Purpose.Attributes.update_forward_refs(**localns)
 Collection.Attributes.update_forward_refs(**localns)
 Folder.Attributes.update_forward_refs(**localns)
+AlteryxWorkflow.Attributes.update_forward_refs(**localns)
```

Comment:
This is a one-off customization (in-progress experiment) — we should remove from generated code.